cluster/ci/monitoring/prometheus_rule_prow.yaml

```diff
@@ -205,3 +205,30 @@ spec:
       for: 1m
       labels:
         severity: slack
+  - name: ghproxy
+    rules:
+    - alert: ghproxy-status-code-abnormal-4XX
+      annotations:
+        message: ghproxy has a increasing number of status code 4XX.
+      expr: |
+        sum(rate(github_request_duration_count{status=~"4.."}[5m])) > 0
+      for: 1m
+      labels:
+        severity: slack
+    - alert: ghproxy-status-code-abnormal-5XX
+      annotations:
+        message: ghproxy has a increasing number of status code 5XX.
+      expr: |
+        sum(rate(github_request_duration_count{status=~"5.."}[5m])) > 0
+      for: 1m
+      labels:
+        severity: slack
+    - alert: ghproxy-running-out-github-tokens-in-a-hour
```

Comment:
Just like the PVC one checks both current capacity and linear prediction, we should -- alert if we're below 30% or something _and_ predicted to run out

Reply:
Totally forgot it.

`1/3` is easier.